scss/product/components/_avatar.scss

```diff
@@ -27,6 +27,13 @@
 	}
 }
 
+.avatar-square{
+	border-radius: 0;
+	background: transparent;
+}
+.no-bg{
+	background:none;
+}
```

Comment:
what's the use of this class?

Reply:
It's not being used currently. Probably it was used earlier. I have removed unused css and pushed the commit